### PR TITLE
PMM-5581 investigate possibility of updating Grafana plugins

### PR DESCRIPTION
**Pull request by adivinho**
- [ ] https://github.com/Percona-Lab/pmm-submodules/pull/754

### CODE REVIEW

File: import-dashboards.py [hunk 1]
```diff
@@ -237,6 +237,22 @@ def add_panels():
...
+
+
+def rename_panels():
+    for app in ['panels']:
+        print '  * Renaming %r' % (app,)
+        panels_dir = '/var/lib/grafana/plugins/'
+        panels_list = os.listdir(panels_dir)
+        for panel in panels_list:
+            print '   * %r -> ' % (panel),
+            panel_path = panels_dir + panel + '/dist/plugin.json'
```
Comment (askomorokhov) [inline import-dashboards.py]: IMHO better to use `os.path.join` here.

File: import-dashboards.py [hunk 2]
```diff
@@ -237,6 +237,22 @@ def add_panels():
...
+            rename_panels()
+
+
+def rename_panels():
+    for app in ['panels']:
+        print '  * Renaming %r' % (app,)
+        panels_dir = '/var/lib/grafana/plugins/'
+        panels_list = os.listdir(panels_dir)
+        for panel in panels_list:
+            print '   * %r -> ' % (panel),
```
Comment (askomorokhov) [inline import-dashboards.py]:
```suggestion
            print '   * %r -> ' % (panel,)
```
It is safer to use tuple here.

File: import-dashboards.py [hunk 3]
```diff
@@ -237,6 +237,22 @@ def add_panels():
...
+            panel_path = panels_dir + panel + '/dist/plugin.json'
+            if os.path.exists(panel_path):
+                with open(panel_path, 'r') as f:
+                    panel_params = json.loads(f.read())
+                    print '\'%s\'' % (panel_id['id'])
```
Comment (askomorokhov) [inline import-dashboards.py]:
```suggestion
                    print '%r' % (panel_id['id'],)
```
May be ?

File: import-dashboards.py (hunk 2 above)
Comment (AlekSi) [reply import-dashboards.py]: btw, @adivinho, trailing comma is only required with there is one element:
```python
(one)       # single value with extra/bogus parentheses 
(one,)      # a tuple with a single value
(one, two)  # a tuple with two values
```

It is required there, but not in other PR where it leads to confusion.

**Pull request merged**